menu/cakes/cakeA.html

```diff
@@ -0,0 +1,269 @@
+<!DOCTYPE HTML>
+<html>
+
+<head>
+  <title>Cake A — Mandy B. Bakes</title>
+  <meta property="og:title" content="Mandy B Bakes" />
+  <!-- <meta property="og:image" content="images/bg.jpg" /> -->
+  <meta property="og:description" content="Amanda B's Online Bakery" />
+  <meta property="og:url" content="https://yu-huanwu.github.io/mandybbakes/" />
+  <meta charset="utf-8" />
+  <meta name="viewport" content="width=device-width, initial-scale=1, user-scalable=no" />
+  <link rel="stylesheet" href="../../assets/css/main.css" />
+  <link rel="icon" type="image/x-icon" href="../../images/favicon.ico" />
+</head>
+
+<body class="is-preload">
+
+  <!-- Wrapper -->
+  <div id="wrapper">
+
+    <!-- Main -->
+    <div id="main">
+      <div class="inner">
+
+        <!-- Header -->
+        <header id="header">
+          <a href="../../index.html" class="logo"><strong>Mandy B. Bakes</strong></a>
+          <ul class="icons">
+           <li><a href="https://instagram.com/mandyb.bakes" class="icon brands fa-instagram"><span
+                  class="label">Instagram</span></a></li>
+          </ul>
+        </header>
+
+        <!-- Banner -->
+        <section id="banner">
+          <span class="image image-left object">
+            <img src="../../images/cakes/cake_pic03.jpg" alt="Amanda next to two-tiered wedding cake" />
+          </span>
+          <div class="content content-right">
+            <header>
+              <h1>Cake A</h1>
+            </header>
+            <h2>Description:</h2>
```

Comment:
Actually do you feel like we need this? Isn't the text under the <h1> heading implicitly the description? up to you. I know the sample websites we reference has it but perhaps ours with <h2> is a bit too big, and honestly it feels redundant anyway.

Reply:
just wanted to follow up: I kept the <h1> in for now but resized both the <h1> and <h2> for these product detail pages.